charts/enterprise-metrics/templates/querier-dep.yaml

```diff
@@ -94,6 +94,8 @@ spec:
               mountPath: /etc/enterprise-metrics
             - name: runtime-config
               mountPath: /var/enterprise-metrics
+            - name: graphite-schemas
```

Comment:
does the querier need this?

Reply:
You're right this was a mistake :+1: Addressed in https://github.com/grafana/helm-charts/pull/872/commits/56157c4b966ca1359833ce0ac8e56562437033f3